src/glue.rs

```diff
@@ -201,7 +202,14 @@ extern "C" {
      -> *const ::libc::c_void;
     pub fn CreateWrapperProxyHandler(aTraps: *const ProxyTraps)
      -> *const ::libc::c_void;
+    pub fn CreateServoJSPrincipal(origin: *const ::libc::c_void,
+                                  destroy: Option<*const ::libc::c_void>,
+                                  write: Option<bool>)
```

Comment:
The destroy and write arguments don't match the C types. See the ones in ProxyTypes for good models.